src/frontends/paddle/src/op/unique.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs unique(const NodeContext& node) {
+    auto x = node.get_input("X");
+
+    std::vector<Output<Node>> outputs;
+    NamedOutputs named_outputs;
+
+    auto axis = node.get_attribute<std::vector<int32_t>>("axis");
+
+    auto dtype_str = node.get_attribute<std::string>("dtype");
+    auto dtype = dtype_str == "int32" ? element::i32 : element::i64;
+
+    if (axis.size() != 0) {
+        auto axis_node = std::make_shared<default_opset::Constant>(element::i32, Shape{}, axis);
+        outputs = std::make_shared<ov::opset10::Unique>(x, axis_node, true, dtype, dtype)->outputs();
+    } else {
+        outputs = std::make_shared<ov::opset10::Unique>(x, true, dtype, dtype)->outputs();
+    }
+
+    named_outputs["Index"] = {outputs[2]};
+    named_outputs["Indices"] = {outputs[1]};
+    named_outputs["Counts"] = {outputs[3]};
```

Comment:
- In OpenVINO unique spec, `outputs[3]` type is controled by `count_element_type`. https://docs.openvino.ai/2023.1/openvino_docs_ops_movement_Unique_10.html.
- In PaddlePaddle spec, I don't find the `return_counts` dtype. I am worried because I run into the similar issue before. Could you confirm it here? I

Reply:
In paddle, the `count_element_type` and `index_element_type` both be controlled by [attr_dtype](https://github.com/PaddlePaddle/Paddle/blob/release/2.5/python/paddle/tensor/manipulation.py#L2514-L2522)